src/compositor/lipstickkeymap.cpp

```diff
@@ -0,0 +1,76 @@
+/***************************************************************************
+**
+** Copyright (C) 2015 Jolla Ltd.
+** Contact: Aaron Kennedy <aaron.kennedy@jollamobile.com>
```

Comment:
Having into account that I'm a Jolla subcontractor and this is under Jolla's copyright, I think it makes sense to keep the same contact person than in other files which are also under Jolla's copyright.

I can replace it with myself in any case, though.